src/apache/conf/httpd.conf

```diff
@@ -206,6 +206,13 @@ ServerSignature Off
     ServerSignature On
 </IfDefine>
 
+# Enable notify_push reverse proxy
+<IfDefine EnableNotifyPushReverseProxy>
+    ProxyPass /push/ws ws://127.0.0.1:7867/ws
+    ProxyPass /push/ http://127.0.0.1:7867/
+    ProxyPassReverse /push/ http://127.0.0.1:7867/
```

Comment:
Quickly perusing the notify_push README leads me to believe this port is entirely up to the user, right? Looks like maybe they need to setup a service that listens on this port? Is it worth making that configurable? Also, if we're going to restrict this to localhost, we might also consider restricting it to a unix socket instead of a port.

Reply:
> Quickly perusing the notify_push README leads me to believe this port is entirely up to the user, right?

I think so

> Looks like maybe they need to setup a service that listens on this port?

Totally. See https://github.com/nextcloud-snap/nextcloud-snap/wiki/Configure-HPB-client-push-for-Nextcloud-snap#b-install-system-deamon

> Is it worth making that configurable?

I don't really see the benefit of making it configurable, to be honest

> If not, if we're going to restrict this to localhost, we might also consider restricting it to a unix socket instead of a port.

That's a great idea, but I don't know how it can be done...

Reply:
Yeah, didn't know socket path is possible... But how to get sich a socket path and how to setup that in Apache for the lines above? Never did that and never seen that.

Reply:
Something like that? Do I need to do something else with it or just use it?
`redis+unix:///tmp/snap-private-tmp/snap.nextcloud/tmp/sockets/notify_push.sock`

Reply:
I'm sorry @Pilzinsel64 , I know that we are also using a socket for the nextcloud <-> MySQL communication, but I don't really know the details of how it works, or how we should use one in this case...

Reply:
Yeah I would try copying what we do for the PHP reverse proxy. Operating off of memory, I believe it looks like what you have there, @Pilzinsel64. Give it a shot and see if it works!

Reply:
Oh dear... two hours of try-and-error later I got a configuration that worked on another production instance (without snap). Let's see if the configuration works here too, because I had trouble getting apache to see my socket in /temp/... while my tests.

The only thing I really wonder is how the paths now should look like. For Apache **inside** the snap I now use the "fake" path to the socket and for the [service configuration](https://github.com/nextcloud-snap/nextcloud-snap/wiki/Configure-HPB-client-push-for-Nextcloud-snap/_compare/19ff9c0eb2e32d1515e7ada0b8c36ad1ee0f6b68...5442930a348d36211d83d34156fc47fc762073da) **outside** the snap I used the "real" path.

@kyrofa can you please double check the socket paths? Should the path itself correct like that?

As soon as the build successed I will try the snap on my test VM.